tests/test_connection.py

```diff
@@ -39,10 +39,8 @@
 
 
 @pytest.fixture
-def event_loop():
```

Comment:
What about the `owner` fixture down below that uses `event_loop`?

Reply:
pytest-asyncio provides the `event_loop` fixture. It just doesn't want you to override it any more as of 0.23 (doing so is deprecated). It also doesn't want you to directly depend on the fixture in async tests, but it's okay in synchronous fixtures.